src/sonic-containercfgd/containercfgd/containercfgd.py

```diff
@@ -0,0 +1,198 @@
+import os
+import re
+import shlex
+import signal
+import subprocess
+import sys
+
+from sonic_py_common import daemon_base, logger
+from swsscommon.swsscommon import ConfigDBConnector, RestartWaiter
+
+SYSLOG_IDENTIFIER = "containercfgd"
+logger = logger.Logger(SYSLOG_IDENTIFIER)
+
+# Table names
+FEATURE_TABLE = 'FEATURE'
+SYSLOG_CONFIG_FEATURE_TABLE = 'SYSLOG_CONFIG_FEATURE'
+
+# Table field names
+SYSLOG_RATE_LIMIT_INTERVAL = 'rate_limit_interval'
+SYSLOG_RATE_LIMIT_BURST = 'rate_limit_burst'
+
+# Container name
+container_name = None
+
+
+def run_command(command):
+    """
+    Utility function to run an shell command and return the output.
+    :param command: Shell command string.
+    :return: Output of the shell command.
+    """
+    return subprocess.check_output(shlex.split(command), text=True, stderr=subprocess.PIPE)
```

Comment:
No need to catch exceptions here?

Reply:
Currently, I would like to let caller to handler the exception, but I am not sure if it is best practice. Any suggestion?

Reply:
I think it's ok to let the caller to handle the exception in this case.